### PR TITLE
update cloud description and small edits

### PR DESCRIPTION
**Pull request by abuehrle**
Pushed these changes again (it's easier)

### CODE REVIEW

File: site/installing.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Weave Cloud is the recommended option if:
  * You are deploying to larger clusters.
  * You require secure remote access.
  * You want to share access with your coworkers.
+ * You want to minimize Weave Scope memory and CPU usage.
```
Comment (2opremio) [inline site/installing.md]: _You want to minimize your local consumption CPU and memory consumption._ would be clearer

Comment (rade) [inline site/installing.md]: I am not so sure. "local" could be mistaken for "where i run my browser".

Comment (2opremio) [inline site/installing.md]: Then how about about plain  _You want to minimize memory and CPU usage._ ?

Comment (rade) [inline site/installing.md]: But it's the memory and CPU usage of *Scope* that users of the standalone Scope observe as being high.

Comment (2opremio) [inline site/installing.md]: Alright

**Comment by 2opremio** [comment]
minor comment, otherwise LGTM

File: site/installing.md [hunk 2]
```diff
@@ -48,43 +49,45 @@ Where,
 
 ###<a name="docker-weave-cloud"></a>Using Weave Cloud
 
-First, obtain a service-token from Weave Cloud by signing up at [https://cloud.weave.works](https://cloud.weave.works/).
+First, obtain a Cloud service token by signing up at [https://cloud.weave.works](https://cloud.weave.works/).
```
Comment (abuehrle) [inline site/installing.md]: How about 'cloud token'? That's what I call it in the guides.

**Comment by 2opremio** [comment]
Sure, cloud token works too, as long as all references are consistent.

**Comment by 2opremio** [comment]
Merging.

@abuehrle Will you do the honors and push the documents with wordepress?

**Pull request merged**